tests/conftest.py

```diff
@@ -80,8 +80,14 @@ def mock_login_token_response():
     mock_token_res.by_resource_server = {
         "auth.globus.org": _mock_token_response_data(
             "auth.globus.org",
-            "openid profile email "
-            "urn:globus:auth:scope:auth.globus.org:view_identity_set",
+            " ".join(
+                [
+                    "openid",
+                    "profile",
+                    "email",
+                    "urn:globus:auth:scope:auth.globus.org:view_identity_set",
+                ]
+            ),
```

Comment:
Technically an unrelated and non-semantic change, I decided to leave this in because it reads better this way.

It made sense to be touching these lines when we added (and then removed) the `view_consents` scope.